internal/scorecard/plugins/plugin_runner.go

```diff
@@ -91,134 +92,47 @@ func RunInternalPlugin(pluginType PluginType, config BasicAndOLMPluginConfig, lo
 		}
 	}()
 
-	var (
-		tmpNamespaceVar string
-		err             error
-	)
+	var tmpNamespaceVar string
+	var err error
 	kubeconfig, tmpNamespaceVar, err = k8sInternal.GetKubeconfigAndNamespace(config.Kubeconfig)
 	if err != nil {
 		return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("failed to build the kubeconfig: %v", err)
 	}
+
 	if config.Namespace == "" {
 		config.Namespace = tmpNamespaceVar
 	}
-	scheme := runtime.NewScheme()
-	// scheme for client go
-	if err := cgoscheme.AddToScheme(scheme); err != nil {
-		return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("failed to add client-go scheme to client: (%v)", err)
-	}
-	// api extensions scheme (CRDs)
-	if err := extscheme.AddToScheme(scheme); err != nil {
-		return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("failed to add failed to add extensions api scheme to client: (%v)", err)
-	}
-	// olm api (CSVs)
-	if err := olmapiv1alpha1.AddToScheme(scheme); err != nil {
-		return scapiv1alpha1.ScorecardOutput{}, fmt.Errorf("failed to add failed to add oml api scheme (CSVs) to client: (%v)", err)
-	}
-	dynamicDecoder = serializer.NewCodecFactory(scheme).UniversalDeserializer()
-	// if a user creates a new CRD, we need to be able to reset the rest mapper
-	// temporary kubeclient to get a cached discovery
-	kubeclient, err := kubernetes.NewForConfig(kubeconfig)
+
+	err = setupRuntimeClient()
```

Comment:
Minor nit to decrease LoC and limit this `err` to the scope of the conditional block:

```go
if err := setupRuntimeClient(); err != nil {
	return scapiv1alpha1.ScorecardOutput{}, err
}

Reply:
fixed this one.